src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -200,16 +200,6 @@ public static String formatDate(Date date) {
         sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
         return sdf.format(date);
     }
-
-    /**
-     * Formats a date in the format dd MMM yyyy, HH:mm. Example: 05 May 2012,
-     * 22:04<br />
-     */
-    public static String formatTime(Date date) {
-        if (date == null)
-            return "";
-        return new SimpleDateFormat("EEE, dd MMM yyyy, HH:mm").format(date);
```

Comment:
If i'm not wrong, it will be UTC.
In any case, for feedback sessions, the time is not adjusted to the timezone specified by the instructor when it is saved, so it is independent of time zone.